### PR TITLE
Include feedhenry tests in build

### PR DESCRIPTION
**Pull request by dgolovin**
Fix contains:
1. feedhenry inclsion in build
2. Changes for  naming consistency
3. Features for cordovasim and feedhenry tests
4. Updated site/category.xml to include test features and test features sources

### CODE REVIEW

**Comment by maxandersen** [comment]
I applied #119 to at least have the build publish results/pass basic tests. This issue should be applied asap in master but I can't verify it here.

File: feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<packaging>eclipse-test-plugin</packaging>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<explodedBundles combine.children="append">
+						<bundle>org.jboss.tools.feedhenry.test</bundle>
```
Comment (maxandersen) [inline feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml]: why is this necessary to customize at this level ? doesn't the bundle itself just need to have manifest stating it wants to be exploded?

**Comment by ibuziuk** [comment]
@maxandersen have tested the PR locally. Build Success. All tests passed
+ 1 to apply

**Comment by kmarmaliykov** [comment]
+1, I review this PR, everything is fine, all-tests and root pom builds ok, build should become green

**Comment by maxandersen** [comment]
did the generated updatesites also contain the new artifacts and you could install from them ?

**Comment by kmarmaliykov** [comment]
@maxandersen yes- I can see necessary site and features after local build

File: feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml (hunk 1 above)
Comment (dgolovin) [inline feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml]: It should have manifest entry about bundle shape, I'll check and send PR for that.

Comment (dgolovin) [inline feedhenry/tests/org.jboss.tools.feedhenry.test/pom.xml]: It can be removed for now, if for some reason test bundles should be exploded after installation use 

``` shell
Eclipse-BundleShape: dir
```

in bundle's MANIFEST.MF file.

PR #20 removes that part of configuration from cordovasim and feedhenry tests.
